android/src/app/organicmaps/settings/SettingsPrefsFragment.java

```diff
@@ -151,9 +147,6 @@ private void setLanguage(@NonNull LanguageData lang)
 
   private void updateTts()
   {
-    if (mPrefEnabled == null || mPrefLanguages == null || mLangInfo == null || mLangInfoLink == null)
-      return;
-
```

Comment:
`updateTts()` should be called only when we initialize TTS settings. We expect no nulls here.

Reply:
Maybe add a comment about it to the code?

Reply:
Added a comment to `updateTts()` and `initTtsLangInfoLink()` methods